docs/configuration/outbox-event-router.asciidoc

```diff
@@ -0,0 +1,195 @@
+= Outbox Event Router
+:awestruct-layout: doc
+:linkattrs:
+:icons: font
+:source-highlighter: highlight.js
+
+[NOTE]
+====
+This single message transformation (SMT) is available since Debezium version 0.9.3.
+It is under active development right now, so the emitted message structure or other details may still change as development progresses.
+====
+
+[NOTE]
+====
+This SMT does *not* support the MongoDB connector.
+====
+
+The Outbox Event Router SMT has the intent to provide out-of-the-box support for the Outbox Pattern.
```

Comment:
> out-of-the-box support for the Outbox Pattern.

Loving that pun ); wanted to suggest it as a title for the blog post already :)

Reply:
Haha, it's impossible not to do this pun!